apps/widget/package.json

```
@@ -69,7 +69,7 @@
     "@types/react-router-dom": "^5.1.7",
     "craco-antd": "^1.19.0",
     "cross-env": "^7.0.3",
-    "cypress": "10.11.0",
+    "cypress": "12.3.0",
```

```suggestion
    "cypress": "^12.3.0",
```
So we both get it updated (web and widget) and just one single version in the dependency all the time.